### ISSUE TITLE
stopping restarting after refresh error.

### ISSUE THREAD

**Issue by xekoukou** [jacomyal/sigma.js#647]
Here is what I get:
```
Uncaught DataCloneError: Failed to execute 'postMessage' on 'Worker': An ArrayBuffer is neutered and could not be cloned.
```

The relevant code:
```
                xgraphs[key].refresh();
                xgraphs[key].stopForceAtlas2();
                xgraphs[key].startForceAtlas2();
```

**Comment by Yomguithereal** [comment]
Hello @xekoukou. Can you tell me on which browser you got this error. Then, can you try the following?:

```js
xgraphs[key].refresh();
xgraphs[key].stopForceAtlas2();

setTimeout(function() {
  xgraphs[key].startForceAtlas2();
}, 0);
```

**Comment by xekoukou** [comment]
Google Chrome:
Version 44.0.2403.156 (64-bit)

Chrome OS

Your fix didn't work.

**Comment by xekoukou** [comment]
I made a demo here:
http://xekoukou.github.io/simul/

**Comment by Yomguithereal** [comment]
It seems you sometimes inject several edges with the same id within your graph and this causes multiple errors. Can you try to fix this and tell me whether this fixes the whole thing or not please?

**Comment by xekoukou** [comment]
That may be the case. Will look into it.

**Comment by xekoukou** [comment]
The problem persists after fixing this.

**Comment by Yomguithereal** [comment]
Ok. Could you try `killForceAtlas2()` instead of `stopForceAtlas2()` please?

**Comment by xekoukou** [comment]
It works flawlessly with ```killForceAtlas2()```

**Comment by Yomguithereal** [comment]
Cool to know it works now @xekoukou. If you want to know the reasons why, you can read the explanation in #504.

**Comment by xekoukou** [comment]
Ok. You can close this issue then if you want. I will wait for a fix of issue #504.
